packages/sdk/src/sdkBase.ts

```diff
@@ -788,25 +791,28 @@ export class NxtpSdkBase {
           return data;
         }),
       );
+
+      const validBids = receivedBids.filter((x) => typeof x !== "string") as AuctionResponse[];
+      const invalidBids = receivedBids.filter((x) => typeof x === "string") as string[];
+
+      if (validBids.length === 0) {
+        throw new NoValidBids(transactionId, payload, invalidBids.join(","), receivedBids);
```

Comment:
This error also needs to be `isAuctionError = true` right?

Reply:
it should inherit that property